conda/pytorch_cnn.sub

```diff
@@ -0,0 +1,25 @@
+universe = vanilla
+executable = pytorch_cnn.sh
+output = $(Cluster)_$(Process).out
+
+transfer_input_files = environment.yml, ../docker/pytorch_python/main.py, ../docker/pytorch_python/MNIST_data.tar.gz 
```

Comment:
I have mixed feelings about this organization - would it make sense to have the data in a standalone folder?

Reply:
I agree it's clunky.  Should we make a top-level `resources` or `shared` directory?

Reply:
I think a `shared` directory makes sense.

Reply:
+1

Reply:
Updated to use `shared` directory in 61f2fc0a05b7eec4363b014c2ea5d891934afc7e  Is this what you had in mind?

I confirmed the conda PyTorch example still runs after these changes.  The Docker PyTorch example is in the queue.

Reply:
looks good!